docs/components/SegmentedControl/SegmentedControl.stories.mdx

```diff
@@ -0,0 +1,111 @@
+import { Icon } from "@jobber/components/Icon";
+
+<Meta
+  title="Components/Selections/SegmentedControl"
+  component={SegmentedControl}
+/>
+
+# SegmentedControl
+
+SegmentedControl is a horizontal control made up of multiple segments or
+options, which adjusts the presentation of a single view. A common use case is
+for swapping between Day/Week/Month views in a calendar. The same content is
+displayed, however the presentation changes based on the selected segment.
+
+## Design & usage guidelines
+
+SegmentedControl should only use up to five options with succinct labels. The
+component will also grow quite wide with more options and can become unusable on
+smaller screens. In that case, it is advised to put the SegmentedControl in a
+parent container with a fixed width. Be sure to avoid truncating the labels.
+
+## Content guidelines
+
+Concise [Text](../?path=/docs/components-text-and-typography-text--docs) should
+be used as the primary format for option labels.
+
+<Canvas>
+  <SegmentedControl defaultOption="option1">
+    <SegmentedControl.Option value="option1">Option 1</SegmentedControl.Option>
+    <SegmentedControl.Option value="option2">Option 2</SegmentedControl.Option>
+    <SegmentedControl.Option value="option3">Option 3</SegmentedControl.Option>
+    <SegmentedControl.Option value="option4">Option 4</SegmentedControl.Option>
+    <SegmentedControl.Option value="option5">Option 5</SegmentedControl.Option>
+  </SegmentedControl>
+</Canvas>
+
+Using [Icons](../?path=/docs/components-images-and-icons-icon--docs) as labels
+should only be used if the icon has been tested as understandable in isolation.
+
+<Canvas>
+  <SegmentedControl defaultOption="calendar">
+    <SegmentedControl.Option value="calendar">
+      <Icon name="calendar" />
+    </SegmentedControl.Option>
+    <SegmentedControl.Option value="phone">
+      <Icon name="phone" />
+    </SegmentedControl.Option>
+    <SegmentedControl.Option value="availability">
+      <Icon name="availability" />
+    </SegmentedControl.Option>
+  </SegmentedControl>
+</Canvas>
+
+Mixing content types within a single label, like using an icon with text, should
+be avoided. This would be an inefficient use of the SegmentedControl's limited
+horizontal space.
+
+Mixing content types across options should also be avoided. This would create an
+inconsistent experience for users and lead to a jarring visual rhythm.
+
+If you do not have enough horizontal room to allow for readable labels, consider
+using [Menu](/?path=/docs/components-navigation-menu--docs) as an alternative
+single-selection method for modifying the view. Otherwise, truncation will
+prevent the user from reading the label and a label with wrapped text will
+create layout inconsistencies.
+
+## Controlled and Uncontrolled
+
+Use an
+[uncontrolled](../?path=/story/components-selections-segmentedcontrol-web--basic-uncontrolled)
+version when the SegmentedControl can manage it's own state. This is useful when
```

Comment:
Given my surface-level understanding, this appears to be explaining the concepts of controlled and uncontrolled components, not how they relate to this component. Is there anything specific to how SegmentedControl can be used in either fashion that users would need to know?

Reply:
Great question - similar to Aiden's above where he pointed out I wasn't explaining how to implement controlled vs uncontrolled for this component specifically. I updated that doc section here 2caca5d47ba6fec9e469e27187d5f5e5e97d48bf